vizro-ai/docs/pages/user-guides/customize-vizro-ai.md

```diff
@@ -1,108 +1,105 @@
 
-When specifying a model as a string, you can select any option from the [supported models](#supported-models) listed above.
+<!-- vale off -->
+To ensure a deterministic answer to our queries, we've set the temperature to 0 in the string instantiation. If you prefer more creative (but potentially more unstable) responses, you can raise the temperature to a maximum of 1.
+<!-- vale on -->
 
-The example below uses the OpenAI model name in a string form:
+Below you can find an example where a custom model is instantiated with various custom parameters. Note that we manually set the API key and base URL, which is sometimes the easiest way to get it setup.
 
-!!! example "Customize with string"
+```py
+import os
+from langchain_openai import AzureChatOpenAI
 
-    === "python"
-        ```py linenums="1"
-        from vizro_ai import VizroAI
+llm = AzureChatOpenAI(
+    azure_deployment="gpt-4-1106-preview",
+    api_version="2024-04-01-preview",
+    temperature=0.4,
+    azure_endpoint=os.environ["AZURE_OPENAI_ENDPOINT"],
+    api_key=os.environ["AZURE_OPENAI_API_KEY"]
+)
 
-        vizro_ai = VizroAI(model="gpt-4o")
-        ```
+vizro_ai = VizroAI(model=llm)
+```
 
-The example below customizes the `ChatOpenAI` instance further beyond the chosen default from the string instantiation. We pass the `"gpt-4o"` model from OpenAI as `model_name` for `ChatOpenAI`, which offers improved response accuracy, we also want to increase maximum number of retries.
-It's important to mention that any parameter that could be used in the `openai.create` call is also usable in `ChatOpenAI`. For more customization options for `ChatOpenAI`, refer to the [LangChain ChatOpenAI docs](https://api.python.langchain.com/en/latest/chat_models/langchain_openai.chat_models.base.ChatOpenAI.html)
+Passing an instantiated model to `VizroAI` lets you customize it, and additionally, it enables you to use a model or vendor that is not included in the above list of [string model names](#setting-model-via-string-for-ease-of-use).
 
-<!-- vale off -->
-To ensure a deterministic answer to our queries, we've set the temperature to 0. If you prefer more creative (but potentially more unstable) responses, you can raise the temperature to a maximum of 1.
-<!-- vale on -->
+## What model to choose?
 
-!!! example "Customize with instantiated model"
+### Chart generation
 
-    === "python"
-        ```py linenums="1"
-        from vizro_ai import VizroAI
-        from langchain_openai import ChatOpenAI
+At the time of writing, we found that for chart creation, some of the leading vendor's "cheaper" models, for example OpenAI's `gpt-4o-mini` and `gpt-3.5` model series, have lower price points and faster speeds, and are sufficient for basic charting.
```

Comment:
lower than what and faster than what? I think maybe a slight revision, see suggestion but feel free to modify...